server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -578,6 +578,16 @@ public final class IndexSettings {
         Property.InternalIndex
     );
 
+    /**
+     * Setting to enable/disable desc leaf reader optimization
+     */
+    public static final Setting<Boolean> SEARCH_SEGMENTS_REVERSE_ORDER_OPTIMIZATION_SETTING = Setting.boolSetting(
+        "index.search_segments_reverse_order_optimization",
```

Comment:
Minor, just to follow the conventions for on/off features:

```suggestion
        "index.search_segments_reverse_order_optimization.enabled",
```

Reply:
yeah , makes sense